lesson_26/api/java/api_app/src/main/java/com/codedifferently/lesson26/web/CreateMediaItemRequest.java

```diff
@@ -13,5 +14,6 @@
 @Builder
 public class CreateMediaItemRequest {
   @NotNull(message = "item is required") @Valid
+  @JsonProperty("item")
```

Comment:
Shouldn't need this annotataion here or in `MediaItemRequest`. Please remove.

Reply:
Done